examples/p4est_2d_dgsem/elixir_navierstokes_SD7003airfoil.jl

```diff
@@ -0,0 +1,150 @@
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the compressible Euler equations
+
+gamma = 1.4
+
+U_inf = 0.2
+aoa = 4 * pi / 180
+rho_inf = 1.4 # with gamma = 1.4 => p_inf = 1.0
+
+Re = 10000.0
+airfoil_cord_length = 1.0
+
+t_c = airfoil_cord_length / U_inf
+
+prandtl_number() = 0.72
+mu() = rho_inf * U_inf * airfoil_cord_length / Re
+
+equations = CompressibleEulerEquations2D(gamma)
+equations_parabolic = CompressibleNavierStokesDiffusion2D(equations, mu = mu(),
+                                                          Prandtl = prandtl_number(),
+                                                          gradient_variables = GradientVariablesPrimitive())
+
+@inline function initial_condition_mach02_flow(x, t, equations)
+    # set the freestream flow parameters such that c_inf = 1.0 => Mach 0.2
+    rho_freestream = 1.4
+
+    v1 = 0.19951281005196486 # 0.2 * cos(aoa)
+    v2 = 0.01395129474882506 # 0.2 * sin(aoa)
```

Comment:
Is this done for speed to avoid the repeated `sin` and `cos` evaluations?
```suggestion
    # Values correspond to `aoa = 4 * pi / 180`
    v1 = 0.19951281005196486 # 0.2 * cos(aoa)
    v2 = 0.01395129474882506 # 0.2 * sin(aoa)
```